streaming/src/main/scala/org/apache/gearpump/streaming/executor/Executor.scala

```diff
@@ -64,10 +64,11 @@ class Executor(executorContext: ExecutorContext, userConf : UserConfig)  extends
     }
 
   def appMasterMsgHandler : Receive = {
-    case LaunchTask(taskId, taskContext, taskClass) => {
+    case LaunchTask(taskId, taskContext, taskClass, taskActorClass) => {
       LOG.info(s"Launching Task $taskId for app: ${appId}, $taskClass")
+      val task = new TaskWrapper(taskClass, taskContext, userConf)
       val taskDispatcher = context.system.settings.config.getString(Constants.GEARPUMP_TASK_DISPATCHER)
-      val task = context.actorOf(Props(taskClass, taskContext, userConf).withDispatcher(taskDispatcher), "group_" + taskId.groupId + "_task_" + taskId.index)
+      val taskActor = context.actorOf(Props(taskActorClass, taskContext, userConf, task).withDispatcher(taskDispatcher), "group_" + taskId.groupId + "_task_" + taskId.index)
     }
     case TaskLocations(locations) =>
       val result = locations.flatMap { kv =>
```

Comment:
I noticed that this class's ackage name is not consistent with directory structure.